CaptureActivity/build.gradle

```diff
@@ -13,6 +13,43 @@ android {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.txt'
         }
+        debug {
+            debuggable = true
+        }
+        flavorDimensions 'std'
+    }
+
+    flavorDimensions 'std'
+    productFlavors.all {
+        buildConfigField "String", "API_BASE_URL", '"http://demo.openimis.org/rest/"'
+        buildConfigField "String", "RAR_PASSWORD", '")(#$1HsD"'
+    }
+    productFlavors {
+        demoProd {
+            buildConfigField "String", "API_BASE_URL", '"http://demo.openimis.org/rest/"'
```

Comment:
Do we need API_BASE_URL in this library?

Reply:
I deleted unused build config fields.